platform/assets/js/app.js

```diff
@@ -66,6 +66,15 @@ Hooks.ScrollToTop = {
     }
 }
 Hooks.InfiniteScroll = InfiniteScroll;
+Hooks.PasteFromClipboard = {
```

Comment:
You probably don't need hooks here. Instead, you can use Alpine JS's `x-on:paste` directive on the comment box.